universal-application-tool-0.0.1/app/views/LoginForm.java

```diff
@@ -52,7 +52,7 @@ public Content render(Http.Request request, Optional<String> message) {
 
     // "defense in depth", sort of - this client won't be present in production, and this button
     // won't show up except when running locally.
-    if (request.host().startsWith("localhost:")) {
+    if (request.host().startsWith("localhost:") || request.host().startsWith("civiform:")) {
```

Comment:
Thoughts (non-blocking): since we have this check in at least two places, maybe we can make it configurable somewhere. Not sure if that's secure though.

Reply:
Done.